doc/authorization.md

```diff
@@ -0,0 +1,165 @@
+   Identities can belong to one or more groups.
+   Permissions can be assigned to groups.
+   TLS clients cannot currently be assigned to groups.
+
+(permissions)=
+### Explore permissions
+
+To discover available permissions that may be assigned to a group, or view permissions that are currently assigned, run
+
+    lxc auth permission list --max-entitlements 0
+
+The entity type column displays the LXD API resource type, this value is required when adding a permission to a group.
+
+The URL column displays the URL of the LXD API resource.
+
+The entitlements column displays all available entitlements for that entity type.
+If any groups are already assigned permissions on the API resource at the displayed URL, they are listed alongside the entitlements that they have been granted.
+
+Some useful permissions at a glance:
+
+- The `admin` entitlement on entity type `server` gives full access to LXD.
+  This is equivalent to an unrestricted TLS client or Unix socket access.
+- The `project_manager` entitlement on entity type `server` grants access to create, edit, and delete projects, and all resources belonging to those projects.
+  However, this permission does not allow access to server configuration, storage pool configuration, or certificate/identity management.
+- The `operator` entitlement on entity type `project` grants access to create, edit, and delete all resources belonging to the project against which the permission is granted.
+  Members of a group with this permission will not be able to edit the project configuration itself.
+  This is equivalent to a restricted TLS client with access to the same project.
+- The `user` entitlement on entity type `instance` grants access to view an instance, pull/push files, get a console, and begin a terminal session.
+  Members of a group with this entitlement cannot edit the instance configuration.
+
+```{note}
+Due to a limitation in the LXD client, if `can_exec` is granted to a group for a particular instance, members of the group will not be able to start a terminal session unless `can_view_events` is additionally granted for the parent project of the instance.
+We are working to resolve this.
+```
+
+(identities)=
+### Explore identities
+
+To discover available identities that may be assigned to a group, or view identities that are currently assigned, run:
+
+    lxc auth identity list
+
+The authentication method column displays the method by which the client authenticates with LXD.
+
+The type column displays the type of identity.
+Identity types are a superset of TLS certificate types and additionally include OIDC clients.
+
+The name column displays the name of the identity.
+For TLS clients, this will be the name of the certificate as shown in the certificates API.
+For OIDC clients this will be the name of the client as given by the IdP (requested via the [profile scope](https://openid.net/specs/openid-connect-basic-1_0.html#Scopes)).
```

Comment:
```suggestion
For OIDC clients this will be the name of the client as given by the :abbr:`IdP (identity provider)` (requested via the [profile scope](https://openid.net/specs/openid-connect-basic-1_0.html#Scopes)).
```